### DISCUSSION TITLE
Downgrading transmission

### DISCUSSION THREAD

**Discussion by racemus** [haugene/docker-transmission-openvpn#2871] [Q&A]
Hi everyone,

I'm not very experienced with Docker, so I need some help. I want to use the latest version of the image but downgrade Transmission to version 3.00. Could anyone explain how to do this?

**Comment by pkishino** [comment]
This doesn’t make any sense.. first of all why? And if you really want an older version just use an older image

**Reply by racemus** [reply]
> first of all why?

Because Transmission 4 doesn't have queue order sorting, which I use

> And if you really want an older version just use an older image

Of course, I want other things updated

**Reply by pkishino** [reply]
You mean this?
https://github.com/transmission/transmission/pull/6753
it’s coming..

Otherwise you are free to build your own image based on this and change the transmission version

**Reply by racemus** [reply]
Yes, it may come in a year or two or perhaps three :))
> Otherwise you are free to build your own image based on this and change the transmission version

As I mentioned, I'm not experienced with Docker. I’d really appreciate some help, or at least a good manual for beginners

**Reply by benyjr** [reply]
I appreciate you can't build your own custom image (neither can I), but asking the maintainer to create a super niche downgraded version for you is highly unreasonable.
